data/component-design-tokens/checkbox-design-tokens.json

```
@@ -73,19 +72,19 @@
     "type": "color"
   },
   "checkbox-color-label": {
-    "value": "{color.text.default}",
+    "value": "{input.color.text.default}",
```

Label uses input.color.text.secondary in design. 

Should use `input.color.text.secondary` (dark grey `#353535`) not `input.color.text.default` (black `#000000`).


```suggestion
    "value": "{input.color.text.secondary}",
```